apps/gql/schema/schema.graphql

```diff
@@ -284,3 +285,15 @@ type RemovePanoUpvotePayload {
   node: PanoUpvote
   error: PanoUpvoteError
 }
+
+### Rafine
+
+type RafineQuery {
+  metadata(input: String!): RafineMetadataResult
+}
+
+type RafineMetadataResult {
```

Comment:
No need to have a "Result" suffix in the typename. `RafineMetadata` is good enough.
```suggestion
type RafineMetadata {
```